src/universal/subscriptions/NotificationSubscription.js

```diff
@@ -1,35 +1,54 @@
-import {showWarning} from 'universal/modules/toast/ducks/toastDuck'
-import {addOrgMutationNotificationUpdater} from 'universal/mutations/AddOrgMutation'
-import {addTeamMutationNotificationUpdater} from 'universal/mutations/AddTeamMutation'
-import {approveToOrgNotificationUpdater} from 'universal/mutations/ApproveToOrgMutation'
+import {
+  addOrgMutationNotificationOnNext,
+  addOrgMutationNotificationUpdater
+} from 'universal/mutations/AddOrgMutation'
+import {
+  addTeamMutationNotificationOnNext,
+  addTeamMutationNotificationUpdater
+} from 'universal/mutations/AddTeamMutation'
+import {
+  approveToOrgNotificationOnNext,
+  approveToOrgNotificationUpdater
+} from 'universal/mutations/ApproveToOrgMutation'
 import {cancelApprovalNotificationUpdater} from 'universal/mutations/CancelApprovalMutation'
 import {cancelTeamInviteNotificationUpdater} from 'universal/mutations/CancelTeamInviteMutation'
 import {clearNotificationNotificationUpdater} from 'universal/mutations/ClearNotificationMutation'
-import {createTaskNotificationUpdater} from 'universal/mutations/CreateTaskMutation'
+import {
+  createTaskNotificationOnNext,
+  createTaskNotificationUpdater
+} from 'universal/mutations/CreateTaskMutation'
 import {deleteTaskNotificationUpdater} from 'universal/mutations/DeleteTaskMutation'
 import handleAddNotifications from 'universal/mutations/handlers/handleAddNotifications'
-import {inviteTeamMembersNotificationUpdater} from 'universal/mutations/InviteTeamMembersMutation'
-import {rejectOrgApprovalNotificationUpdater} from 'universal/mutations/RejectOrgApprovalMutation'
-import getInProxy from 'universal/utils/relay/getInProxy'
+import {
+  inviteTeamMembersNotificationOnNext,
+  inviteTeamMembersNotificationUpdater
+} from 'universal/mutations/InviteTeamMembersMutation'
+import {
+  rejectOrgApprovalNotificationOnNext,
+  rejectOrgApprovalNotificationUpdater
+} from 'universal/mutations/RejectOrgApprovalMutation'
 import toTeamMemberId from 'universal/utils/relay/toTeamMemberId'
-import {removeOrgUserNotificationUpdater} from 'universal/mutations/RemoveOrgUserMutation'
+import {
+  removeOrgUserNotificationOnNext,
+  removeOrgUserNotificationUpdater
+} from 'universal/mutations/RemoveOrgUserMutation'
 
 const subscription = graphql`
   subscription NotificationSubscription {
     notificationSubscription {
       __typename
-      ...AddOrgMutation_notification
-      ...AddTeamMutation_notification
-      ...ApproveToOrgMutation_notification
-      ...CancelApprovalMutation_notification
-      ...CancelTeamInviteMutation_notification
-      ...ClearNotificationMutation_notification
-      ...CreateTaskMutation_notification
-      ...DeleteTaskMutation_notification
-      ...InviteTeamMembersMutation_notification
-      ...RemoveOrgUserMutation_notification
-      ...RejectOrgApprovalMutation_notification
-      ...UpdateUserProfileMutation_notification
+      ...AddOrgMutation_notification @relay(mask: false)
```

Comment:
This is so you always get all the fields?

Reply:
yeah, IMO this is a bug in relay. I think masking should automatically be turned off for mutations & subscriptions.
without this, the result in the `onNext` handler does not contain any of those fragments.